techniques/general/G220.html

```diff
@@ -95,27 +95,27 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
 		
 		
 		<section id="description">
-			<h2>Description</h2>
-			<p>The objective of this technique is to provide a mechanism for finding contact details in a consistent location across pages to make it easier for users to find it. The interactive item in the web page is a link to the contact details page. The programmatic and visual location is consistent on each page in the set of web pages, when viewed within the same size viewport. Activating the link brings users to a web page with contact details such as a phone number and/or email address.</p>
+			<h2>解説</h2>
+			<p>このテクニックの目的は、ページ間で一貫した場所で連絡先情報を見つけるメカニズムを提供して、利用者が簡単に見つけられるようにすることである。ウェブページ内のインタラクティブな項目は、連絡先情報ページへのリンクである。プログラム的及び視覚的な位置は、同じサイズのビューポート内で閲覧したときに、ウェブページ一式内の各ページで一貫している。リンクをクリックすると、電話番号、電子メールアドレスなどの連絡先情報が記載されたウェブページに移動する。</p>
```

Comment:
1文目、原文の意味としては "The objective ... is to provide a mechanism" なので、目的は「見つけられるようにすること」ではなく「メカニズムを提供すること」と言っているように思います。

2文目、なんか唐突感がありますね。原文からして唐突に The interactive item が登場していて、これ、もしかして The mechanism の間違いなんじゃなかろうかと思いつつ……。
「ウェブページ内にインタラクティブな項目があり、連絡先情報ページへのリンクとなっている。」
というくらいに訳しても良いように思いました。

こんな感じでどうでしょうか……?

```suggestion
			<p>このテクニックの目的は、利用者が連絡先情報を簡単に見つけられるように、ページ間で一貫した場所で連絡先情報を見つけるメカニズムを提供することである。ウェブページ内にインタラクティブな項目があり、連絡先情報ページへのリンクとなっている。プログラム的な位置及び視覚的な位置は、同じサイズのビューポートで閲覧している限り、ウェブページ一式内の各ページで一貫している。リンクをクリックすると、電話番号、電子メールアドレスなどの連絡先情報が記載されたウェブページに移動する。</p>
```